browser/src/control/jsdialog/Util.Dropdown.js

```diff
@@ -151,6 +154,12 @@ JSDialog.OpenDropdown = function (id, popupParent, entries, innerCallback, popup
 	};
 
 	L.Map.THIS.fire('jsdialog', {data: json, callback: generateCallback(entries)});
+	const listContainer = JSDialog.GetDropdown(id);
+	if(useListKeyboardNavigation && listContainer) {
+		// attach list keybinding for arrow-key navigation in entry list
+		JSDialog.KeyboardListNavigation(listContainer);
```

Comment:
I think this can be moved to dropdown handler which doesn't exist currently (it does just default job).
That's why we see in the browser console: `JSDialogBuilder: Unsupported control type: "dropdown"` after we open dropdown.

Please try to add new `JSDialog.Dropdown` handler function which will just do the build based on received JSON and this KeyboardListNavigation if needed.

it should be something like:
```
JSDialog.Dropdown = function(parentContainer, data, builder) {
    builder.build(parentContainer, data.children, data.vertical === true);

   if (!data.noDefaultKeyboardNavigation)
         JSDialog.KeyboardListNavigation(parentContainer);
    return false;
}
```
and regiser it in the JSDialogBuilder: `this._controlHandlers['dropdown'] = JSDialog.Dropdown;`
and colorpicker should have in dropdown JSON property: `noDefaultKeyboardNavigation: true`

then we shouldn't need that code here

Reply:
> colorpicker should have in dropdown JSON property: `noDefaultKeyboardNavigation: true`

@eszkadev  I think we do not need this `noDefaultKeyboardNavigation` because color picker already has handler: `this._controlHandlers['colorpicker'] = JSDialog.colorPicker;`

Thanks for the Dropdown handler approach i think it's awesome :)